proposals/2025/Add Federal Fields.md

```diff
@@ -0,0 +1,117 @@
+# Add Federal Fields
+
+## Overview
+
+This proposal is to add new elements under the `FederalBuilding` element:
```

Comment:
Maybe add a note here that the `FederalBuilding` element is being added under `Building`?